lib/loopback.js

```diff
@@ -76,6 +95,10 @@ function createApplication() {
   app.connector('memory', loopback.Memory);
   app.connector('remote', loopback.Remote);
 
+  // setup the app registry
+  var registry = app.registry = new Registry();
+  require('./builtin-models')(registry);
+
```

Comment:
Ha, I noticed now that the flag is a known TODO item :)

> Add a flag that sets app.registry as loopback.registry for true backwards compatability

The part "don't include builtin-models in the local registries" is still relevant.